### DISCUSSION TITLE
How do I set the dtmf method to info ?

### DISCUSSION THREAD

**Discussion by boxerab** [baresip/baresip#2547] [General]
it seems that default is in-band

**Comment by juha-h** [comment]
Please read Wiki pages first and ask if answer is not found there,

**Comment by boxerab** [comment]
@juha-h thanks, I took a look. I see that both info and 4733 is supported. One confusing issue - on the Android baresip client, the options are in-band and info - do you know if `in-band` means in fact 4733 ?

**Comment by juha-h** [comment]
Aaron Boxer writes:

> @juha-h thanks, I took a look. I see that both info and 4733 is
> supported. One confusing issue - on the Android baresip client, the
> options are in-band and info - do you know if `in-band` means in fact
> 4733 ?

The text is "In-band RTP events" and it does mean RFC 4733.  I don't
think there exists any third way to transmit DTMF digits between parties
of a call.

**Reply by boxerab** [reply]
in-band used to mean sending DTMF directly in audio, that's why I was confused. Now I understand, thanks.
